src/platform/site-wide/sass/style.scss

```diff
@@ -1,10 +1,10 @@
 /*
- * NOTE: there is a version of this file for brand consolidation at
- * brand-consolidate.scss. If you make a change to this file, please
+ * NOTE: there is a version of this file for the regular vets.gov build at
+ * style.scss. If you make a change to this file, please
  * verify that you do or do not need to make the same change in the other file
  */
 @import "modules/m-overrides";
-@import "~@department-of-veterans-affairs/formation/sass/core";
+@import "~@department-of-veterans-affairs/formation/sass/core.scss";
```

Comment:
Interesting... why is the suffix needed here?

Reply:
The diff here is sorta misleading. This `style.scss` is ultimately just `brand-consolidation.scss` but renamed. I didn't change its contents. GitHub is showing the diff between this and the original `style.scss`.